components/information-modal.html

```diff
@@ -0,0 +1,23 @@
+<div class="flex flex-col justify-center min-h-screen pt-4 bg-gray-100 min-w-screen justify-center items-center">
+
+    <div class="rounded-lg shadow-lg bg-white my-3 max-w-sm overflow-x-auto">
+        <div class="flex border-b border-gray-100 px-5 py-4">
+            <div class="flex items-center justify-center w-5">
+                <svg class="w-6 h-6 text-white fill-current text-blue-500" viewBox="0 0 40 40"
+                     xmlns="http://www.w3.org/2000/svg">
+                    <path d="M20 3.33331C10.8 3.33331 3.33337 10.8 3.33337 20C3.33337 29.2 10.8 36.6666 20 36.6666C29.2 36.6666 36.6667 29.2 36.6667 20C36.6667 10.8 29.2 3.33331 20 3.33331ZM21.6667 28.3333H18.3334V25H21.6667V28.3333ZM21.6667 21.6666H18.3334V11.6666H21.6667V21.6666Z"/>
+                </svg>
+            </div>
+            <span class="font-bold text-gray-700 text-lg ml-2">Information modal</span>
+        </div>
+
+        <div class="px-10 py-5 text-gray-600">
+            Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua.
+        </div>
+
+        <div class="px-5 py-4 flex justify-end">
```

Comment:
Maybe another background for the footer section just to show the distinction of separate sections.

```suggestion
        <div class="bg-gray-100 border-t border-gray-300 px-5 py-4 flex justify-end">
```